servicecontrol/containerization/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Running ServiceControl in containers
+reviewed: 2020-12-10
+---
+
+Docker images for ServiceControl exist on Dockerhub under the [Particular organization](https://hub.docker.com/u/particular). These can be used to run ServiceControl in docker containers. These docker images are only available for Windows.
+
+## Containers overview
+
+ServiceControl is split into multiple docker images for each instance type. These are:
+
+* The error instance
+* The audit instance
+* The monitoring instance
+
+The images for each of these containers are further split into an init container and a runtime container. This means that there are 6 images for all instances of ServiceControl.
+
+The init container is used to create or upgrade the infrastructure required for ServiceControl. This includes creating the database as well as the queues that ServiceControl uses. Once the init container has been run, it will shut down and the runtime container can be run. The runtime container will use the infrastructure that has been created.
+
+### Transports
+
+Each supported transport and topology for ServiceControl has further been broken down into it's own set of images. The following transport and topologies are available on dockerhub:
+
+* [Amazon SQS](https://hub.docker.com/search?q=servicecontrol.amazonsqs&type=image)
+* [Azure Service Bus](https://hub.docker.com/search?q=servicecontrol.azureservicebus&type=image)
+* [Azure Storage Queues](https://hub.docker.com/search?q=servicecontrol.azurestorageque&type=image)
+* [RabbitMQ Conventional Topology](https://hub.docker.com/search?q=servicecontrol.rabbitmq.conven&type=image)
+* [RabbitMQ Direct Topology](https://hub.docker.com/search?q=servicecontrol.rabbitmq.direct&type=image)
+* [Sql Server](https://hub.docker.com/search?q=servicecontrol.sqlserver&type=image)
+
+## Running ServiceControl using Docker
+
+This section uses the SqlServer transport as an example on how to run ServiceControl using docker. The same steps are applicable to other transports.
+
+Each instance type of ServiceControl has a distinct init and runtime container.
+
+* The error instance of ServiceControl can be run as stand-alone
+* The audit instance needs an error instance to be running too
+* The monitoring instance can be run as stand-alone
+
+### Error instances
+
+The first step is to create the required infrastructure using the `init` container:
+
+`docker run -e "ServiceControl/ConnectionString=[connectionstring]" -e 'ServiceControl/LicenseText=[licensecontents]' -v c:/data/:c:/data/ -v c:/logs/:c:/logs/ -d particular/servicecontrol.sqlserver.init-windows:4`
```

Comment:
It's not entirely clear this connectionstring var is also used with Azure Service Bus. Maybe mention that explicitely. And explain all available vars.

Reply:
Added a section explaining that all variables are environment variables